src/components/SidePanel/TriggerSettings.tsx

```diff
@@ -98,12 +112,37 @@ export default () => {
                 onChange={setInternalTriggerLength}
                 onChangeComplete={(value: number) => {
                     dispatch(setTriggerRecordingLength(value));
+                    setComputedBias(calculateBiasTime(value, triggerBias));
                 }}
                 unit="ms"
                 label="Length"
                 disabled={dataLoggerActive}
                 showSlider
             />
+            <NumberInput
+                range={{
+                    min: 0,
+                    max: 100,
+                    decimals: 0,
+                    step: 1,
+                }}
+                title='Trigger bias from "Start of trigger window"'
+                value={triggerBiasValue}
+                onChange={setTriggerBiasValue}
+                onChangeComplete={(value: number) => {
+                    dispatch(setTriggerBias(value));
+                    setComputedBias(
+                        calculateBiasTime(internalTriggerLength, value)
+                    );
+                }}
+                unit="%"
+                label="Bias"
+                disabled={dataLoggerActive}
+                showSlider
+            />
+            <span className="tw-mb-2 tw-text-sm tw-text-gray-500">
+                Computed bias: {computedBias} ms
+            </span>
```

Comment:
If this renders under the slider for the bias i would  prefer a div tag instead of span. It is more of a. personal preference so feel free to ignore.

I associate span tag with inline text that needs some special formatting hence the above suggestion